autogen.sh

```diff
@@ -120,7 +120,7 @@ for PARAM; do
 done
 
 if test x$has_ext_mod = xtrue; then
-	pushd ../mono-extensions/scripts
+	pushd $top_srcdir../mono-extensions/scripts
```

Comment:
shouldn't that be `$top_srcdir/../mono-extensions/scripts`?

Reply:
No, `$top_srcdir` is always `/`-terminated.